docs/getting-started/prerequisites-wsl2.md

```diff
@@ -0,0 +1,212 @@
+# Setting up the prerequisites with WSL2
+
+This guide describes how to install the prerequisites required to deploy
+Carbyne Stack via WSL2 in windows.
+
+!!! warning
+    Carbyne Stack has been tested using the **exact versions** of the tools
+    specified below. Deviating from this _battle tested_ configuration may
+    create all kinds of issues.
+
+## Prerequisites
+
+    WSL2 must be installed in your Windows machine.
+
+    To install WSL2, run your Windows powershell and run this command
+
+    ```shell
+    wsl --install
+    ```
+
+Or you can refer to https://www.windowscentral.com/how-install-wsl2-windows-10 if more details are needed.
+
+## Platform Setup Prerequisites
+
+Software to be installed:
+
+### install Ubuntu-20.04 inside WSL2 and connect to it
+    ```shell
+    wsl install Ubuntu-20.04\
+    wsl -d Ubuntu-20.04
+    ```
+
+### Configure DNS Resolution
+    1. prevent creating the resolve.conf on startup
+    ```shell
+    sudo nano /etc/wsl.conf
+    ```
+    And add this text to wsl.conf file
```

Comment:
Use

> And add this text to `wsl.conf` file

Reply:
Please check that you adhere to this style for filenames throughout the whole file